src/collections/_documentation/platforms/python/index.md

```diff
@@ -2,55 +2,398 @@
 title: Python
 ---
 
-{% include learn-sdk.md platform="python" %}
+## Integrating the SDK
 
-The [Sentry Python SDK](https://pypi.org/project/sentry-sdk) provides support
-for Python 2.7 and 3.4 or later.
+Sentry captures data by using an SDK within your application’s runtime. These are platform specific and allow Sentry to have a deep understanding of how your application works.
 
-This documentation goes over some Python specific things such as integrations to
-frameworks.
+Install our Python SDK using `pip`:
 
-## Integrations
+```bash
+$ pip install --upgrade sentry-sdk
+```
 
-*Integrations* extend the functionality of the SDK for some common frameworks and
-libraries. Similar to plugins, they extend the functionality of the Sentry
-SDK. A call to `sentry_sdk.init` configures integrations. Unless you set `default_integrations` to `False`, Sentry automatically adds any default integration not in the list below.
+### Connecting the SDK to Sentry
 
-### Web Frameworks
+After you’ve completed setting up a project in Sentry, Sentry will give you a value which we call a DSN or Data Source Name. It looks a lot like a standard URL, but it’s just a representation of the configuration required by the Sentry SDKs. It consists of a few pieces, including the protocol, public key, the server address, and the project identifier.
 
-* [Django]({% link _documentation/platforms/python/django.md %})
-* [Flask]({% link _documentation/platforms/python/flask.md %})
-* [Sanic]({% link _documentation/platforms/python/sanic.md %})
-* [Pyramid]({% link _documentation/platforms/python/pyramid.md %})
-* [AIOHTTP]({% link _documentation/platforms/python/aiohttp.md %})
-* [Tornado]({% link _documentation/platforms/python/tornado.md %})
-* [Bottle]({% link _documentation/platforms/python/bottle.md %})
-* [Falcon]({% link _documentation/platforms/python/falcon.md %})
-* [Generic WSGI]({% link _documentation/platforms/python/wsgi.md %})
+Import and initialize the Sentry SDK early in your application’s setup:
 
-### Task Queues
+```python
+import sentry_sdk
```

Comment:
Do we want to keep the dynamic DSN placeholders we have e.g. here? https://docs.sentry.io/error-reporting/quickstart/?platform=javascript

If so we should replace the DSN with `___DSN___`

Reply:
We probably should keep them wherever we use the DSN in configuration examples.